main.py

```diff
@@ -1,19 +1,24 @@
-import http.client
-import json
+from reader import HttpRequestResponse
 import datetime
 from util import SortUtil
 
-print('Start')
+#get current time
 start = datetime.datetime.now()
-conn = http.client.HTTPSConnection('api.spaceflightnewsapi.net')
-headers = {'Content-type': 'application/json'}
-conn.request("GET", "/v3/articles?_limit=30", headers=headers)
-response = conn.getresponse()
-js = json.loads(response.read().decode())
+
+#create instance of a class
+getData = HttpRequestResponse('api.spaceflightnewsapi.net', '/v3/articles?_limit=30')  
```

Comment:
- Avoid naming an object 'getSomething'. Read about naming object in OOP.
- Use snake_case.

Reply:
object naming has been changed